src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -167,7 +167,7 @@
     examples:
         - name: Change the cluster durability level to 'Silver'.
           text: >
-            az sf cluster durability update -g group-name -n cluster1 --durability-level Silver
+            az sf cluster durability update -g group-name -n cluster1 --durability-level Silver --node-type 'ns1'
```

Comment:
Why the single quotes on ns1? They aren't necessary.

Reply:
They are unnecessary, indeed. i used them, because I copied other examples.